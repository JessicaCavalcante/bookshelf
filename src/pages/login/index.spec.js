import { Login } from './index.js';
// import { googleLogin } from '../../services/index.js';
// import { emailAndPasswordLogin } from '../../services/index.js';

describe('Login', () => {
  it('should be a function', () => {
    expect(typeof Login).toBe('function');
  });

  it('should load the page', () => {
    expect(Login()).toMatchSnapshot();
  });
});

// describe('emailAndPasswordLogin', () => {
//   it('should be a function', () => {
//     expect(typeof renderPage).toBe('function');
//   });
// });