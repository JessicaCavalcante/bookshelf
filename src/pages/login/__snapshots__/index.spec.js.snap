// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should load the page 1`] = `
<div
  class="container"
>
  
      
  <header>
    
        
    <h2
      class="title-login"
    >
      Bookshelf
    </h2>
    
      
  </header>
  
      
  <section>
    
        
    <form
      class="form-login"
    >
      
          
      <label
        class="label-enter"
      >
        Entrar
      </label>
      
          
      <label
        class="label-login"
      >
        E-mail: 
        <span
          class="required"
        >
          *
        </span>
      </label>
      
          
      <input
        class="input-login"
        id="email-login"
        name="email-login"
        required=""
        type="email"
      />
      
          
      <label
        class="label-login"
      >
        Senha: 
        <span
          class="required"
        >
          *
        </span>
      </label>
      
          
      <input
        class="input-login"
        id="password-login"
        minlength="8"
        name="password-login"
        required=""
        type="password"
      />
      
          
      <button
        class="btn-login"
        id="btn-login"
        type="submit"
      >
        Entrar
      </button>
      
          
      <label
        class="alternative"
      >
        Ou
      </label>
      
          
      <button
        id="btn-google"
      >
        
            
        <img
          alt="Ícone do Google"
          class="btn-google-icon"
          src="../../img/google-icon.svg"
        />
        
            
        <span
          class="btn-google-text"
        >
          Entre com conta Google
        </span>
        
          
      </button>
      
        
    </form>
    
      
  </section>
  
      
  <p
    class="sign-up-login"
  >
    
        Ainda não tem uma conta? 
    <a
      href="#"
      id="sign-up-login"
    >
      Registrar-se
    </a>
    
      
  </p>
  
  
</div>
`;
